examples/upload/upload.html

```diff
@@ -8,7 +8,7 @@
     <h1>Storj Browser Upload Example</h1>
     <input type="file" id="input" onchange="handleFiles(this.files)">
     <script>
-      var bucketId = '77845a36aadcb966fc76d5da';
+      var bucketId = '24213949f1400a77cfcbd287';
```

Comment:
We really need to setup a demo cluster somewhere for our examples. @phutchins possibly a good use for flexing the `storj-sdk` with some pseudo-production traffic and surfacing some bugs there? Wouldn't need anything too crazy, just a single VM with ~1GB of RAM (for `npm install`) and maybe 20GB of HDD space.